packages/fast-components-styles-msft/src/action-toggle/index.ts

```diff
@@ -73,23 +90,28 @@ const styles: ComponentStyles<ActionToggleClassNameContract, DesignSystem> = {
         },
         "&$actionToggle__disabled $actionToggle_selectedGlyph, &$actionToggle__disabled $actionToggle_unselectedGlyph": {
             fill: accentForegroundRest,
+            ...applyHighContrastDisabledFill(),
         },
     },
     actionToggle__justified: {
         "& $actionToggle_selectedGlyph, & $actionToggle_unselectedGlyph": {
             fill: accentForegroundRest,
```

Comment:
It seems like there is a lot of duplication here - I'm concerned about how much code this is adding for a marginal use-case.

I'm wondering if we can reduce some of the duplication by using `!important`

Reply:
@nicholasrice  would like to sync with you more regarding this approach.

Reply:
Made a pass to eliminate duplicated HC media queries, where possible. Surprising there we a lot, especially in our buttons.